examples/custom-resources/jwt/README.md

```diff
@@ -67,3 +67,21 @@ Date: 10/Sep/2020:18:20:03 +0000
 URI: /
 Request ID: db2c07ce640755ccbe9f666d16f85620
 ```
+
+> **Note**:<br>
+You can add a ``gunzip`` option to the VirtualServer spec. Adding the ``gunzip`` allows NIC to decompress responses where an item
+like a JWT token is compressed by the IdP.<br>
+If an IdP compresses a JWT token and NIC is not configured to decompress responses (``gunzip`` not set to ``on``), the error "invalid JWK set while sending to client" is generated by NIC.<br>
+When the ``gunzip`` value is set to ``on``, NIC automatically decompresses responses with “Content-Encoding: gzip” header.
+
+Example:
+```yaml
+apiVersion: k8s.nginx.org/v1
+kind: VirtualServer
+metadata:
+  name: webapp
+spec:
+  host: webapp.example.com
+  gunzip: on
```

Comment:
Should this be 'true'/'false' now? Same for the *Note* above.
> When the ``gunzip`` value is set to ``on``,

Reply:
`on` has the same meaning as `true` when parsing yaml. [docs](https://nginx.org/en/docs/http/ngx_http_gunzip_module.html#gunzip) refers to values `on` / `off` for the gunzip directive. So, kept the same for NIC naming convention.